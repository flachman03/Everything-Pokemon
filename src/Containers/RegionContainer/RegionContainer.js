import React from 'react'
import { connect } from 'react-redux'
import { RegionCard } from '../../Components/RegionCard/RegionCard'
import './RegionContainer.scss'

export class RegionContainer extends React.Component {
  constructor() {
    super();
    this.state = {}
  }

  render() {
    const { data } = this.props
    const displayData = data.map( region => {
      return <RegionCard region={region} />
    })
    return (
      <section>
        <h1>Find your Region!</h1>
        <ul className="region-list">
        {displayData}
        </ul>
      </section>
    )
  }
}

export default connect(null, null)(RegionContainer)