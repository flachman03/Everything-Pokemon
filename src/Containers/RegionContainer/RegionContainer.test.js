import React from 'react'
import { shallow } from 'enzyme'
import { RegionContainer } from './RegionContainer'

describe('RegionContainer', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<RegionContainer data={[]}/>)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should have a default state', () => {
    const expected = {}
    expect(wrapper.state()).toEqual(expected)
  })
})