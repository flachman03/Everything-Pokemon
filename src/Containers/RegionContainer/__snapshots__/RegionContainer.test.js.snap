// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegionContainer should match the snapshot 1`] = `ShallowWrapper {}`;
